Makefile

```diff
@@ -118,7 +118,7 @@ go.cachedir:
 # NOTE(hasheddan): we must ensure up is installed in tool cache prior to build
 # as including the k8s_tools machinery prior to the xpkg machinery sets UP to
 # point to tool cache.
-build.init: $(UP)
+build.init: $(CROSSPLANE_CLI)
```

Comment:
Q: Is the comment still useful? I don't know the relationship between $(UP) and $(CROSSPLANE_CLI).

Reply:
Good point, will remove the comment - basically the `up` CLI  has been replaced by the `crossplane` CLI